.editorconfig

```
@@ -0,0 +1,5 @@
+root = true
+
+[*]
```

We do not enforce this for any other files (like the README or any CI jobs).
```suggestion
[*.xml]
```